File: PermMissingElem/php/TestPermMissingElem.php
<?php
require 'PermMissingElem.php';

class Test extends PHPUnit_Framework_TestCase {

	protected $perm_missing_elem;

	protected function setup() {
		$this->perm_missing_elem = new \Codility\PermMissingElem;
	}

	public function testSolution() {
		$this->assertEquals(4, $this->perm_missing_elem->solution([2,3,1,5]));
	}


}

File: PermMissingElem/ruby/TestPermMissingElem.rb
require_relative 'PermMissingElem'
require 'minitest/autorun'

class Test < MiniTest::Test

	def setup
		@perm_missing_elem = PermMissingElem.new
	end

	def test_solution
		assert_equal 4, @perm_missing_elem.solution([2,3,1,5])
	end

end
